resources/definitions/helloBEEprusa.def.json

```diff
@@ -10,21 +10,28 @@
         "category": "Other",
         "platform": "BEEVERYCREATIVE-helloBEEprusa.stl",
         "platform_offset": [-226, -75, -196],
-        "file_formats": "text/x-gcode"
+        "file_formats": "text/x-gcode",
+		"machine_extruder_trains":
+        {
+            "0": "hBp_extruder_left",
+            "1": "hBp_extruder_right"
+        }
     },
 
     "overrides": {
+	    "machine_extruder_count": { "default_value": 2 },
         "machine_name": { "default_value": "hello BEE prusa" },
-        "machine_start_gcode": { "default_value": "; -- START GCODE --\nG21                     ;set units to millimetres\nG90                     ;set to absolute positioning\nM107                    ;set fan speed to zero (turned off)\nG28 X0 Y0               ;move to the X/Y origin (Home)\nG28 Z0                  ;move to the Z origin (Home)\nG92 E0                  ;zero the extruded length\nG1 F3600                ;set feedrate to 60 mm/sec\n; -- end of START GCODE --" },
+        "machine_start_gcode": { "default_value": "; -- START GCODE --\nG21                     ;set units to millimetres\nG90                     ;set to absolute positioning\nM107                    ;set fan speed to zero (turned off)\nG28 X0 Y0               ;move to the X/Y origin (Home)\nG28 Z0                  ;move to the Z origin (Home)\nG92 E0                  ;zero the extruded length\nG1 F3600                ;set feedrate to 60 mm/sec\nM420 S1 \n; -- end of START GCODE --" },
         "machine_end_gcode": { "default_value": "; -- END GCODE --\nM104 S0                 ;set extruder temperature to zero (turned off)\nM140 S0                 ;set bed temperature to zero (turned off)\nG28 X0 Y0               ;move to the X/Y origin (Home)\nM84                     ;turn off steppers\n; -- end of END GCODE --" },
         "machine_width": { "default_value": 185 },
         "machine_depth": { "default_value": 200 },
         "machine_height": { "default_value": 190 },
         "machine_heated_bed": { "default_value": true },
         "machine_center_is_zero": { "default_value": false },
-        "material_print_temperature": { "default_value": 220 },
+        "material_print_temperature": { "default_value": 200 },
         "material_bed_temperature": { "default_value": 60 },
         "material_diameter": { "default_value": 1.75 },
+		"line_width": { "default_value": 0.48 },
```

Comment:
Are you sure that a line width of 0.48 with a nozzle size of 0.4 is going to work?

Well, your call I guess :)